src/components/modules/DetailedPage/ProductHeader/ProductHeader.tsx

```diff
@@ -41,7 +44,10 @@ export default function ProductHeader({
           </Row>
         </Column>
         <Column>
-          <Username>{username}</Username>
+          <Row>
+            <UsernameAvatar>{usernameAvatar}</UsernameAvatar>
```

Comment:
I think this needs to be an img, look at the review down.

Reply:
Just fixed this. Placeholder image has been uploaded.